scripts/fast-install.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+@exec('ln -s '.__DIR__.'/../files/data '.__DIR__.'/../web');
```

Comment:
Don't hide errors with `@`: if that link is broken, some features will be broken as well and that must be properly reported. You can even convert errors/warnings to exceptions like in other scripts, that way we'll have a complete stackstrace if the problem happens.